core/net/rpl/rpl-mrhof.c

```diff
@@ -118,6 +120,23 @@ reset(rpl_dag_t *dag)
 }
 
 static void
+dao_ack_callback(rpl_parent_t *p, int status)
```

Comment:
The `dao_ack_callback` will be only called if `RPL_WITH_DAO_ACK` is set. This function is not required for instance if we use the `RPL_MOP_NO_DOWNWARD_ROUTES`. Should we put it between `#if RPL_WITH_DAO_ACK` guards? You will also need to do the same in `rpl-of0.c`.